dandiapi/api/views/asset.py

```diff
@@ -296,17 +333,18 @@ def destroy(self, request, versions__dandiset__pk, versions__version, **kwargs):
         responses={
             200: None,  # This disables the auto-generated 200 response
             301: 'Redirect to object store',
-        }
+        },
+        manual_parameters=[ASSET_ID_PARAM, VERSIONS_DANDISET_PK_PARAM, VERSIONS_VERSION_PARAM],
```

Comment:
Does order matter for these parameters? I would assume `ASSET_ID_PARAM` should be last.

Reply:
Nope, swagger seems to arrange them in the correct order regardless
![Screenshot from 2021-08-05 17-41-39](https://user-images.githubusercontent.com/37340715/128424987-67381c3b-27ee-43ee-bddb-b580faad5b78.png)